### PR TITLE
PLGN-86: add Apple Pay to WooCommerce cardknox plugin

### PR DESCRIPTION
**Pull request by pritesh-lr**

### CODE REVIEW

File: includes/class-wc-cardknox-api.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class WC_Cardknox_API {
 	/**
 	 * Cardknox API Endpoint
 	 */
-	const ENDPOINT = 'https://x1.cardknox.com/gateway';
+	const ENDPOINT = 'https://x3.cardknox.com/gateway';
```
Comment (mstein-ck) [inline includes/class-wc-cardknox-api.php:32]: Why is this needed?

File: assets/js/cardknox-admin.min.js
Comment (mstein-ck) [inline assets/js/cardknox-admin.min.js:1]: Can these changes be made to the un-minified file (cardknox-admin.js) and then minified into this file.

File: assets/js/cardknox.min.js
Comment (mstein-ck) [inline assets/js/cardknox.min.js:1]: Same for this minified file

File: includes/class-wc-cardknox-api.php (hunk 1 above)
Comment (pritesh-lr) [reply includes/class-wc-cardknox-api.php:32]: @mstein-ck , when I apple pay test, it is not working. After that, update the 'https://x3.cardknox.com/gateway' endpoint. It works. also, I checked on the demo server; Magento uses this endpoint.

File: assets/js/cardknox-admin.min.js
Comment (pritesh-lr) [reply assets/js/cardknox-admin.min.js:1]: yes sure, I will do

File: assets/js/cardknox.min.js
Comment (pritesh-lr) [reply assets/js/cardknox.min.js:1]: yes, I will update it.

Comment (pritesh-lr) [reply assets/js/cardknox.min.js:1]: @mstein-ck, I updated js with minify. Please review it.

File: assets/js/cardknox-admin.min.js
Comment (pritesh-lr) [reply assets/js/cardknox-admin.min.js:1]: @mstein-ck, I updated js with minify. Please review it.

File: includes/class-wc-cardknox-api.php (hunk 1 above)
Comment (ybroyde) [reply includes/class-wc-cardknox-api.php:32]: @pritesh-lr Do you have an example of a payment that failed?

Comment (pritesh-lr) [reply includes/class-wc-cardknox-api.php:32]: ApplePay is not working so I updated `https://x3.cardknox.com/gateway` endpoint. but credit card is working fine with `https://x1.cardknox.com/gateway`
